ChaosMod/Effects/db/Misc/MiscSpawnUfo.cpp

```diff
@@ -0,0 +1,334 @@
+/*
+	Author: Kanawanagasaki
+*/
+
+#include <algorithm>
+#include <math.h>
+#include <random>
+#include <stdafx.h>
+
+#include "Memory/Physics.h"
+#include "Util/Text.h"
+
+struct UfoData
+{
+	Object prop;
+	Ped target;
+	Vector3 destination = Vector3();
+	Vector3 prevDestination = Vector3();
+	Vector3 velocity = Vector3();
+	Vector3 acceleration = Vector3();
+	bool isElevating = false;
+	bool isRunningAway = false;
+	bool fadeOut = false;
+	int opacity = 255;
+};
+
+const float UFO_RADIUS = 26.f;
+const int MAX_UFO_AMOUNT = 3;
+const int SPAWN_INTERVAL = 13000;
```

Comment:
```suggestion
static const float UFO_RADIUS = 26.f;
static const int MAX_UFO_AMOUNT = 3;
static const int SPAWN_INTERVAL = 13000;
```
(or just make them macros)